config/routes.rb

```diff
@@ -1,10 +1,12 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  resources :users, only: [:index,:show] do
+    resources :posts, only: [:index,:show,:new, :create]
+  end
   # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
   # Can be used by load balancers and uptime monitors to verify that the app is live.
   get "up" => "rails/health#show", as: :rails_health_check
-
+  root  to: "main#index"
```

Comment:
- [ ] Kindly remove this route, since it unnecessary. You can use the existing users index route as the root URL if that's what you are aiming for.

  ```
  root "users#index"
  ```

Reply:
well resolved! ✔️